test/lib/validation/schema/specVersion/2.0/kind/project/commonUtils.js

```diff
@@ -0,0 +1,494 @@
+const test = require("ava");
+
+test("commonUtils", async (t) => {
```

Comment:
I think we should rather exclude this file in the ava config (see package.json).

Reply:
good point! done